src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall(Ball ball) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        ball.setColor(colorSupplier.getRandomColor());
+        ball.setNumber(new Random().nextInt(100));
+
+        return ball;

```

Comment:
two points:

1. you should define a constant field for `ColorSupplier` so that you don't create new instance of this class each time this method is called. same for `Random`
2. `100` is a magic number - you should create a dedicated contants for specifying the max ball number
3. this method should not receive any parameters - it should return "brand-new" ball as the result. by the way, passing objects as parameters and changing them inside of the method is an anti-pattern, since it leads to unexpected behavior. don't do that in future :)

```suggestion 
    private static final int MAX_BALL_NUMBER = 100;
    private final ColorSupplier colorSupplier = new ColorSupplier();
    private final Random random = new Random();
    
    public Ball getRandomBall() {
        Ball ball = new Ball():
        ball.setColor(colorSupplier.getRandomColor());
        ball.setNumber(random.nextInt(MAX_BALL_NUMBER));

        return ball;

```